tests/validators/blocks/test_list_collector_content_validator.py

```diff
@@ -5,7 +5,11 @@
 
 
 def test_invalid_list_collector_content_for_list():
-    """Tests that when a for_list for a list collector content is not from supplementary data or another list collector it is found to be invalid."""
+    """Test invalid list collector content for list.
```

Comment:
Extra description added but not part of this PR? cause otherwise we would need to add docstrings to every test?